powershell-adapter/Tests/TestClassResource/0.0.1/TestClassResource.psm1

```diff
@@ -15,7 +15,7 @@ class TestClassResource
     [string] $Prop1
 
     [DscProperty()]
-    [EnumPropEnumeration] $EnumProp
+    [string] $EnumProp
```

Comment:
Why was this changed from the enum type?

Reply:
something causes the output to return the index instead of the string. I can't seem to reproduce it in PowerShell. It could be something I am doing in Get-ActualState, or the class for output type, but I haven't been able to find it. It could also be related to how the input/output is handled outside of powershell.